Better descriptions of publicationEvents

tormodv opened the pull request:
And at the same time a few more technical changes in EBUCore due to new tool.

The changes in the CCDM document are more to the point.

JuergenGrupp reviewed (commented):
I agree with the texts in dcterms:desription. 
The rdfs:label elements have identical strings, so planned vs executed is not distinguishable. Would it be helpful to at least add "PublicationEvent start/end date & time as executed"?.

tormodv commented:
Added the changes for CCDM according to your replays. 
And there are many technical changes in EBUCore as part of the process to make this owl file work.
Highlights: Changed a lot of xsd:string containing language resources to Litteral
dc namespace that was meant as annotations were splitter from the propper ec properties. Instead of "Equivalent to" I use rdfs:isDefinedBy.

I think this can be merged now.
